docs/baking.md

```diff
@@ -157,7 +157,7 @@ key that will be used for baking and endorsing.
 
 ### Setting the Liquidity Baking toggle vote option
 
-Since `013-PtJakart`, the `--liquidity-baking-toggle-vote` command line option for
+Since `PtKathma`, the `--liquidity-baking-toggle-vote` command line option for
```

Comment:
I think that we can remove the protocol number, but this option was indeed made mandatory since jakarta, so that shouldn't be changed:
```suggestion
Since `PtJakart`, the `--liquidity-baking-toggle-vote` command line option for
```